content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -275,12 +275,23 @@ When the workloads are deployed, they send CSR requests with related signer info
 
 ## Cleanup
 
-* Remove the `istio-system`, `foo` and `bar` namespaces:
+*   Remove the sample applications `sleep` and `httpbin`:
 
     {{< text bash >}}
-    $ kubectl delete ns istio-system
-    $ kubectl delete ns foo
-    $ kubectl delete ns bar
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
```

Comment:
These changes are no longer cleaning up properly (e.g., the bar ns). It's not breaking the build, because this is an untested document.